compiler/daml-extension/BUILD.bazel

```diff
@@ -103,7 +103,9 @@ genrule(
         sed -i 's/"name": "daml"/"name": "daml-bundled"/' package.json
         $$DIR/$(location //:yarn)
         $$DIR/$(location //:yarn) compile
-        $$DIR/$(location @daml_extension_deps//vsce/bin:vsce) package -o $$DIR/$@
+        # vsce needs Yarn or NPM in path and for some reason NPM
+        # fails to find npm-cli.js so we use Yarn.
```

Comment:
Maybe an instance of https://github.com/bazelbuild/rules_nodejs/issues/1565?

Reply:
Looks like it! Although that issue is pretty old so I would have expected that we hit this before? I’m now slightly worried that this will break publishing to NPM as well :thinking:

Reply:
We hit it before on publishing and fixed it with [this patch](https://github.com/digital-asset/daml/blob/e0b9a535b32b57e9f1a79f0626458e3e1edfcfbc/bazel_tools/rules_nodejs_npm_cli_path.patch) (also linked on the issue). I'm not sure what's different in this case.

Reply:
Alright, then I’m less worried. The workaround of using Yarn seems somewhat sensible here given that we use Yarn everywhere else.